app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +36,8 @@
             android:name=".circle_of_trust.Trustees"
             android:label="@string/title_activity_trustees"
             android:parentActivityName=".MainActivity"
-            android:windowSoftInputMode="stateHidden">
+            android:windowSoftInputMode="adjustPan"
```

Comment:
I haven't fully gone through it. I will let you know when I am done so you can do the changes.